calc/vxlan_resolver.go

```diff
@@ -263,6 +265,48 @@ func (c *VXLANResolver) OnHostConfigUpdate(update api.Update) (_ bool) {
 			})
 			c.sendVTEPRemove(nodeName)
 		}
+	case "VXLANTunnelMACAddr":
+		nodeName := update.Key.(model.HostConfigKey).Hostname
+		vtepSent := c.vtepSent(nodeName)
+		logCxt := logrus.WithField("node", nodeName).WithField("value", update.Value)
+		logCxt.Debug("VXLANTunnelMACAddr update")
+		if update.Value != nil {
+			// Update for a VXLAN tunnel MAC address.
+			newMAC := update.Value.(string)
+			currMAC := c.nodeNameToVXLANMac[nodeName]
```

Comment:
I think you should get `currMAC` by calling `vtepMACForHost()`  Then we'll avoid sending a dupe if the datastore is updated to contain the auto-generated MAC.

Reply:
Acknowledged and changed the code.